synopsys-task/src/synopsys-task/synopsys-bridge.ts

```diff
@@ -73,6 +73,35 @@ export class SynopsysBridge {
     }
 
     try {
+      if (inputs.ENABLE_NETWORK_AIR_GAP) {
+        if (inputs.SYNOPSYS_BRIDGE_PATH.length !== 0) {
```

Comment:
why are we validating like inputs.SYNOPSYS_BRIDGE_PATH.length !== 0, can't we check like if(inputs.SYNOPSYS_BRIDGE_PATH.length) 
is there any specific reason to check empty string with length